addons/index.md

```diff
@@ -7,16 +7,108 @@ layout: documentation
 # Add-ons
 
 All add-ons for openHAB 2 are part of the distribution.
-This includes all new 2.0 bindings as well as all 1.x add-ons that were reported to be compatible.
-Add-ons can be easily installed through the Paper UI (menu entry "Add-ons").
-
-
-| Add-on Type | Description |
-|-------------|----------------------|
-| [Bindings](bindings.html) | Bindings integrate physical hardware, external systems and web services in openHAB |
-| [User Interfaces](uis.html) | User interfaces are either native smartphone apps or web applications that access the openHAB server through the REST API |
-| [Persistence](persistence.html) | Persistence services allow openHAB to store time series data for history-based actions or statistics |
-| [Actions](actions.html) | Actions are predefined methods for openHAB rules and scripts |
-| [Transformations](transformations.html) | Transformations are used to translate between technical and human-readable values for items |
-| [Voice Services](voices.html) | Services that provide voice enabling features, such as text-to-speech, speech-to-text etc. |
-| [3rd Party System Integration](io.html) | Expose openHAB to external systems |
+This includes all 2.x bindings as well as all 1.x add-ons that were reported to be compatible.
```

Comment:
I believe it would be a best practice to always refer to capitalize the words "Items", "Things", "Bindings", and other terms when we are specifically referring to them in the context of an OH "Item", for example.  That way we help the reader to know these are specific things.  It also frees us up as writers to use the lower case term, "thing", for example, when we need to use that term in a sentence because it is the best word for the job, but we are concerned that it will confuse the reader.

This would be a HUGE undertaking to go back and rewrite everything that way, but if, as we are working through documentation, we can all make changes to capitalization as we come across it, then over time, things <- (here is an example ;-) will get better.